client/blocks/signup-form/index.jsx

```diff
@@ -275,13 +275,9 @@ class SignupForm extends Component {
 				? {}
 				: mapKeys( response.messages, ( value, key ) => camelCase( key ) );
 
-			// On Jetpack sign up form, prevent "field is empty" error messages from displaying prematurely
+			// Prevent "field is empty" error messages from displaying prematurely
 			// before the form has been submitted or before the field has been interacted with (is dirty).
-			if (
-				config.isEnabled( 'jetpack/simplify-login-signup-flows' ) &&
-				this.isJetpack() &&
```

Comment:
Initially we began implementing this fix/enhancement as part of project [Simplify Jetpack Account Creation](p1HpG7-bae-p2), therefore we put it behind a feature flag and restricted it to Jetpack connect and Jetpack Cloud sign in only. 
But then after discussion with @rcanepa , we realized that this fix would likely would be a beneficial enhancememt for all applications that use this sign on component, which in addition to Jetpack includes WordPress.com and WooCommerce.
That is why I am now removing this conditional check for `jetpack/simplify-login-signup-flows` feature flag, and `isJetpack()`.